tools/theme-screenshot.js

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/node
+
+let puppeteer
+try {
+  // eslint-disable-next-line import/no-absolute-path
+  puppeteer = require('/home/limon/.npm-packages/lib/node_modules/puppeteer')
```

Comment:
oops, forgot to handle this properly, will do soon

Reply:
done in https://github.com/sweetalert2/sweetalert2.github.io/commit/263d3e512c7f6fdbe7929cf1c179561a38f20a1b